cli.js

```diff
@@ -44,6 +44,14 @@ Usage: $0 <command> [options]` // eslint-disable-line comma-dangle
   .command('create <function> [options]', 'Create a function from template', (yargs0) => {
     yargs0
       .usage('Usage: $0 create <function> [options]')
+      // TODO: should runtime be positional (required)?
+      .option('runtime', {
+        alias: 'r',
+        choices: ['node8', 'cpython2', 'pypy2'],
+        describe: 'Runtime image name',
```

Comment:
`image name`? The term `image` does not belong here. it's leaking the implementation (docker image) and the user doesn't care about it. Should be just `runtime`.